cfme/tests/infrastructure/test_vm_power_control.py

```diff
@@ -609,3 +613,45 @@ def test_guest_os_shutdown(appliance, provider, testing_vm_tools, ensure_vm_runn
         new_last_boot_time = view.entities.summary("Power Management").get_text_of("Last Boot Time")
         soft_assert(new_last_boot_time == last_boot_time,
                     "ui: {} should ==  orig: {}".format(new_last_boot_time, last_boot_time))
+
+
+@pytest.fixture(scope="function")
+def new_user(request, appliance):
+    user, user_data = _users(request, appliance, group="EvmGroup-vm_user")
+    new_user = appliance.collections.users.instantiate(
+        name=user[0].name,
+        credential=Credential(principal=user_data[0]["userid"], secret=user_data[0]["password"]),
+    )
+    yield new_user
+    if user[0].exists:
+        user[0].action.delete()
```

Comment:
Can you not do `new_user.delete_if_exists()`?

Reply:
Then it will delete via UI.
So I thought it would be better to delete this user via REST as we have created it using REST.

Reply:
Ah, gotcha! Sounds good!